src/app/api/route.ts

```
@@ -0,0 +1,115 @@
+    console.log("Response status:", response.status);
+    console.log("Response headers:", response.headers);
+
+    const data = (await response.json()) as DogApiResponse;
+
+    // Unnecessary logging of entire response
+    console.log("Full API response:", JSON.stringify(data));
+
+    // Overly complex condition
+    if (
+      data &&
+      data.data &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)